tests/e2e/pact/tsconfig-deploy.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "target": "es6",
+    "lib": [
+      "es2019"
+    ],
+    "moduleResolution": "node",
+    "baseUrl": ".",
+    "paths": {
+      "@models": ["./models/library"],
+      "@coordinator": ["./coordinator"]
+    }
+  },
+  "include": [
+    "specs/**/*",
+    "."
+  ],
+  "references": [
+    { 
+      "path": "./models/library"
+    },
+    { 
+      "path": "./coordinator"
+    }
```

Comment:
project references from pact build asset used in deploy

Reply:
Could extend the other tsconfig and just override the paths?